### PR TITLE
SYM-1613 - Add ability to attach multiple attachements on client messages

### PR DESCRIPTION
**Pull request by JulienBlacas**
This PR is meant to give the developper the ability to send multiple attachements inside one message

### CODE REVIEW

File: lib/MessagesClient/index.js [hunk 1]
```diff
@@ -27,14 +27,32 @@ MessagesClient.sendMessageWithAttachment = (
...
-    return send(conversationId, message, data, fileName, fileType, fileContent)
+    return send(conversationId, message, data, [{fileName, fileType, fileContent}], sessionToken)
+  }
+};
+
+MessagesClient.sendMessageWithMultipleAttachments = (
+  conversationId,
+  message,
+  data,
+  files,
```
Comment (symphony-adnane) [inline lib/MessagesClient/index.js:46]: does this have any public documentation? 
we should be clair on the format of the files array

**Comment by symphony-adnane** [comment]
Just some open comments 
Thanks Julien for adding this capacity, it's very needed
May you please attach the user story reference to help us understand better the context (it's always good to have some extra info)
I don't see any unit tests on this change, which scares a bit the shit out of me, probably we didn't have it setup already, but we should thing about it
I don't see any way to document the method signatures, we should think about using typescript so we could create definition files automatically and help developers

I'm adding these notes here just so we are aware of these improvements, 
Thanks again

**Review by symphony-adnane — DISMISSED** [review]

**Comment by JulienBlacas** [comment]
@symphony-adnane You are correct, typescript would help a lot in this situation . For the moment, we could think of a validation function that check if the **files** parameter is sent with the proper scheme. If this is not so, we can return an Error which explain how the files array should be structured. For more context about our user story here is the corresponding JIRA ticket : [SYM-1613](https://perzoinc.atlassian.net/browse/SYM-1613)

**Comment by symphony-adnane** [comment]
> @symphony-adnane You are correct, typescript would help a lot in this situation . For the moment, we could think of a validation function that check if the **files** parameter is sent with the proper scheme. If this is not so, we can return an Error which explain how the files array should be structured. For more context about our user story here is the corresponding JIRA ticket : [SYM-1613](https://perzoinc.atlassian.net/browse/SYM-1613)

Sounds good, we'll work midterm on supporting typescript. 
Could you btw add some unit tests for this, I saw that jest is already setup! thanks

**Comment by JulienBlacas** [comment]
@symphony-adnane Yes I will add some unit tests and keep you posted

**Review by symphony-elias — APPROVED** [review]
Looks good ! 
Successfully tested on devx3

**Pull request merged**